realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -1618,6 +1648,68 @@ public void copyToRealmOrUpdate_iterableChildObjects() {
     }
 
     @Test
+    public void copyToRealmOrUpdate_objectInOtherThreadThrows() {
```

Comment:
These 3 tests have too much copy/paste. Isn't it possible to refactor the commonality out?